resources/puppetlabs/puppetdb/pql/pql-grammar-experimental.ebnf

```diff
@@ -30,7 +28,7 @@ extract = <lbracket>, [<whitespace>], [extractfields], [<whitespace>], <rbracket
 <extractfields> = fieldlist;
 
 (* Filtering *)
-<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], [modifier], [<whitespace>], <rbrace>;
+<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], {modifier | pagingclause, [<whitespace>]}, [<whitespace>], <rbrace>;
```

Comment:
Maybe we can actually resolve by requiring paging clauses to come before group by clauses when specified together, but I'm not sure the AST support is there for the combo yet.